feat: [tour] footer visibility fix

etomaello opened the pull request:
Changed the footer style to make it compatible with static image background coming from the hero banner.

Fix #115 

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu
- After: https://fix-115-tour-footer-invisible--realmadrid--hlxsites.hlx.page/sites/tour-bernabeu

vdua commented:
@etomaello It looks fine, but the faq and footer have no gap while it is there in the original website (though the footer is different there)

@chicharr do we have some designs with the new header/footer ?

|Original|Franklin|
|---|---|
|<img width="1255" alt="image" src="https://github.com/hlxsites/realmadrid/assets/2097597/982da06f-0c96-414b-a60d-34f39a66a700">|<img width="1646" alt="image" src="https://github.com/hlxsites/realmadrid/assets/2097597/c993c5ff-e5b7-470b-94c3-d02425e901d3">|

etomaello commented:
> @etomaello It looks fine, but the faq and footer have no gap while it is there in the original website (though the footer is different there)
> 
> @chicharr do we have some designs with the new header/footer ?
> Original 	Franklin
> <img alt="image" width="1255" src="https://user-images.githubusercontent.com/2097597/242157559-982da06f-0c96-414b-a60d-34f39a66a700.png"> 	<img alt="image" width="1646" src="https://user-images.githubusercontent.com/2097597/242157590-c993c5ff-e5b7-470b-94c3-d02425e901d3.png">

Hi @vdua ,
I've modified the style of the footer to make it visible in tour pages without showing any gap between it and the faq, that's why I removed the margin top and added the padding instead, since this ticket doesn't involve the restyling of the block.
As far as I can see RM has the "pre-footer" component that adds extra spacing, I could add that spacing inside the footer block itself if you agree.
